app/src/main/res/values/strings.xml

```diff
@@ -1357,6 +1360,10 @@ You must enter the API key to operate the map normally."</string>
     <string name="design_tab_title_component">Component</string>
     <string name="design_tab_title_event">Event</string>
     <string name="design_tab_title_view">View</string>
+    <string name="developer_tool_component_description_guide_add_component">"If you add a component here
+ You can use it in Logic Editor"</string>
+    <string name="developer_tool_component_message_export">Are you sure you want to export %s ?</string>
```

Comment:
i don't think there should be a space before the `?`, that makes it look like engrish